Multisig/UI/UI Library/BrowseLinkButton.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  BrowseLinkButton.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct BrowseLinkButton: View {
+    @State
+    private var showsLink: Bool = false
+
+    var title: String
+    var url: URL
+    var iconName: String?
+
+    var body: some View {
+        Button(action: {
+            self.showsLink.toggle()
+        }) {
+            HStack {
+                if iconName != nil {
+                    Image(iconName!)
+                }
+                
+                BodyText(title)
+                Spacer()
+                Image(systemName: "chevron.right")
+                    .font(Font.footnote.bold())
+                    .foregroundColor(Color.systemGray6Light)
+            }
+        }
+        .sheet(isPresented: $showsLink, content: browseSafeAddress)
+    }
+
+    func browseSafeAddress() -> some View {
```

Comment:
this one-line function does not add much value here, I would inline the contents with the line 35.